config/locales/instructions/en.yml

```diff
@@ -66,11 +66,9 @@ en:
           PIN</strong> (your PIN was created when you set up your PIV/CAC).
         try_again: try again
       sms:
-        confirm_code_html: Need another code? %{resend_code_link}. Message rates may apply.
```

Comment:
I don't think these have been used since they were removed in https://github.com/18F/identity-idp/pull/2317/files#diff-ae99a10afbc6d44d068d939574f96f9db1db5664be32d11ed56e7ff4bcd6275dL14-L15